### DISCUSSION TITLE
Grouped by category publication page

### DISCUSSION THREAD

**Discussion by jesusfbes** [alshedivat/al-folio#566] [Q&A] [answered]
Hello, 

I am building the publication page and I prefer a layout similar to the one of the author: https://maruan.alshedivat.com/publications/ instead of the standard grouped by year. 

Can anybody help me with this?  I have tried to separate in different bib files but it seem that I must change something in publications.md

**Comment by alshedivat** [comment, marked as answer]
this is a code snippet from `_pages/publications.md` on my personal homepage that could be helpful:
```html
<div class="publications">

<h1>preprints</h1>

{% bibliography -f preprints %}

<h1>conference &amp; journal articles</h1>

{% for y in page.years %}
  <h2 class="year">{{y}}</h2>
  {% bibliography -f papers -q @*[year={{y}}]* %}
{% endfor %}

<h1>technical reports &amp; short papers</h1>

{% bibliography -f reports %}

</div>
```

**Reply by stephenzwj** [reply]
@alshedivat may we know where is your personal github page repo?

**Reply by pdeperth** [reply]
Hello! How do you specify a paper's type in the bib file?

**Reply by pdeperth** [reply]
@jesusfbes else a simple way to achieve this is to modify, in the `_config.yml` file, in the Jekyll Scholar section, the `group_by` parameter to: 
 `group_by: type, year`
